azure/template.json

```
@@ -0,0 +1,163 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "resourceEnvironmentName": {
+            "type": "string",
+            "metadata": {
+                "description": "Short name of the environment. Used for the name of resources created"
+            }
+        },
+        "serviceName": {
+            "type": "string"
+        },
+        "NServiceBusConnectionString": {
```

```suggestion
        "nserviceBusConnectionString": {
```

- Parameter should be camelCase as above.
- Should this be in config to remain consistent with other apps?